<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NonIM</class>
 <widget class="QWidget" name="NonIM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form2</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Non-IM Contact</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="TextLabel1">
        <property name="text">
         <string>First Name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtFirst"/>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel2">
        <property name="text">
         <string>Last Name:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtLast"/>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel1_2">
        <property name="text">
         <string>Nickname:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtNick"/>
      </item>
      <item>
       <widget class="QLabel" name="lblMail">
        <property name="text">
         <string>E-Mail address:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtMail"/>
      </item>
      <item>
       <widget class="QLabel" name="TextLabel4">
        <property name="text">
         <string>Phone:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPhone"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="Spacer7">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
